projects/aca-content/src/lib/components/shared-link-view/shared-link-view.component.html

```diff
@@ -1,9 +1,7 @@
 <ng-container *ngIf="sharedLinkId">
   <adf-alfresco-viewer [allowPrint]="false" [allowDownload]="false" [allowFullScreen]="false" [sharedLinkId]="sharedLinkId" [allowGoBack]="false">
     <adf-viewer-toolbar-actions>
-      <ng-container *ngFor="let action of viewerToolbarActions; trackBy: trackByActionId">
```

Comment:
I think trackByActionId function from shared-link-view.component.ts has no more usages after that change so you can remove that function